autoload/airline/extensions.vim

```diff
@@ -336,6 +336,13 @@ function! airline#extensions#load()
     call add(s:loaded_ext, 'lsp')
   endif
 
+  if (get(g:, 'airline#extensions#nvimlsp#enabled', 1)
+        \ && exists(':LspInstallInfo'))
+        \ && has ('nvim')
```

Comment:
is the `has('nvim')` condition necessary?

Reply:
@crisbra

nvim-lsp's plugin is only moving `neovim`.
Therefore, I think that should write this checking.

Reply:
Yes understood. But if this plugin is neovim specific, than on Vim there should not be a `:LspInstallInfo` command, right? so the check for `:LspInstallInfo` should be sufficient, shouldn't it?

Reply:
Sure, that makes sense.
thanks for your code review.

Reply:
Removed `has('nvim')` condition.